src/content/docs/components/collapsible/readme.mdx

```diff
@@ -4,14 +4,23 @@ The `<Collapsible/>` component allows you to collapse/expand an element with arb
 and apply a nice transition to it. This is a challenge to achieve with CSS only, since it's impossible
 to transition the `height` property from `0` to `auto` and vice versa.
 
-We overcome this challenge by applying a height value based on the `clientHeight` of the content
-and then setting it to `auto` as soon as the transition is completed so that further changes
-in height are automatically applied. This essentially creates a transition between `0` and `auto`.
+This challenge is overcome by applying a *height* value based on the `clientHeight` of the content
+and then set it to `auto` as soon as the transition is complete so that further changes
+in *height* are automatically applied. This essentially creates a transition between `0` and `auto`.
 
 This means that a `<Collapsible/>` is able to expand/collapse from a height of `0` to the height
 of its content, regardless of what the content is, and without any complex calculations.
 You can even nest collapsibles!
 
+The transition *duration* default value is `300ms` and is controlled by the
```

Comment:
Please also add a section called `CSS Custom Properties` under `API` for this property, as we do here: https://webrix.amdocs.com/docs/components/scrollable#examples-css-custom-properties